src/components/Authorization/Authorization.css

```diff
@@ -225,66 +201,43 @@ a{
   }
 }
 
-@media (max-width: 660px) {
-  .eye-icon {
-    position: absolute;
-    top: 32.1%;
-    left: 73%;
+@media (max-width: 600px) {
```

Comment:
We should use all media queries about screen size from one constant (I mean design system file with all constants like colors, containers, typography etc.).

Reply:
fix